core/src/main/scala/cats/data/package.scala

```diff
@@ -58,6 +58,12 @@ package object data {
   type State[S, A] = StateT[Eval, S, A]
   object State extends StateFunctions
 
+  type StoreT[F[_], A, B] = IndexedStoreT[F, A, A, B]
+  object StoreT extends StoreTFunctions
+
+  type StoreId[A, B] = StoreT[Eval, A, B]
```

Comment:
Ideally this should have been called `Store` but it clashes with [this](https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/data/package.scala#L80)

Reply:
I'm really ignorant to this, but are these two `Store` implementations equivalent to each other?

Reply:
It might be, see the discussion below!